public/.htaccess

```diff
@@ -3,16 +3,16 @@
 # mod_rewrite). Additionally, this reduces the matching process for the
 # start page (path "/") because otherwise Apache will apply the rewriting rules
 # to each configured DirectoryIndex file (e.g. index.php, index.html, index.pl).
-DirectoryIndex website.php
```

Comment:
this is just an update of the `.htaccess` from the skeleton version: https://github.com/sulu/skeleton/blob/2.4.1/public/.htaccess